packages/create-cloudflare/src/cli.ts

```diff
@@ -83,8 +83,12 @@ const printBanner = () => {
 	startSection(`Create an application with Cloudflare`, "Step 1 of 3");
 };
 
-export const parseArgs = async (argv: string[]): Promise<Partial<C3Args>> => {
-	const args = await yargs(hideBin(argv))
+const parseArgs = async (argv: string[]): Promise<Partial<C3Args>> => {
+	const doubleDashesIdx = argv.indexOf('--');
+	const c3Args = doubleDashesIdx < 0 ? argv : argv.slice(0, doubleDashesIdx < 0 ? undefined : doubleDashesIdx);
```

Comment:
This is a little redundant, either
```suggestion
	const c3Args = doubleDashesIdx < 0 ? argv : argv.slice(0, doubleDashesIdx);
```
or
```suggestion
	const c3Args = argv.slice(0, doubleDashesIdx < 0 ? undefined : doubleDashesIdx);
```
would do the same thing

Reply:
yeah.... refactoring leftover 😓 

Thanks! 🙂👍